src/main/java/com/RegisterQuizApplication/Registration/QuizController.java

```diff
@@ -0,0 +1,30 @@
+package com.RegisterQuizApplication.Registration;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@CrossOrigin(origins = "*")
+public class QuizController {
+
+    @Autowired
+    QuizRepository repository;
+
+
+
+    //Display the Questions to the User (10 Questions)
+    @GetMapping(path="questions")
+    public List<Quiz> get(
+            @RequestParam(required = false) String field
+    ){
+        if (field==null)
```

Comment:
What if i give somerandom values like abc to the field? The current logic will break.